.github/CODEOWNERS

```
@@ -1,7 +1,17 @@
 # Public packages
-/packages/   @nvie
-/packages/liveblocks-node/   @nvie @ofoucherot @FlowFlorent
-/packages/liveblocks-react-ui/   @marcbouchenoire
+/packages/liveblocks-core/	@nvie @sugardarius @marcbouchenoire
```

I'd prefer to keep the fallback in, but fine with the more specific overloads (later rules will override this one for specific packages):

```suggestion
/packages/   @nvie
/packages/liveblocks-core/	@nvie @sugardarius @marcbouchenoire
```